apps/client/src/app/components/admin-market-data/asset-profile-dialog/asset-profile-dialog.html

```diff
@@ -51,6 +51,28 @@ <h1 class="flex-grow-1 m-0" mat-dialog-title>
       [symbol]="data.symbol"
       (marketDataChanged)="onMarketDataChanged($event)"
     ></gf-admin-market-data-detail>
+
+    <div class="mt-3">
+      <mat-form-field appearance="outline" class="w-100">
+        <mat-label i18n>Historical Data</mat-label>
+        <textarea
+          cdkAutosizeMaxRows="5"
+          cdkTextareaAutosize
+          matInput
+          placeholder="e.g. 20230601;1.61"
+          type="text"
+          [ngModelOptions]="{standalone: true}"
+          [(ngModel)]="historicalDataAsString"
+        ></textarea>
+      </mat-form-field>
+    </div>
+
+    <div class="d-flex justify-content-end" mat-dialog-actions>
+      <button color="warn" mat-flat-button (click)="importHistoricalData()">
```

Comment:
The red color in my mockup is a bit misleading 😄 Can you please use `color="accent"`?

Reply:
screenshot: 
![image](https://github.com/ghostfolio/ghostfolio/assets/11473935/f6bd9181-f62f-4ab6-97bd-37449806d10f)